tools/indextemplate/indextemplate_test.go

```diff
@@ -239,7 +239,7 @@ func TestHandleIndexGood(t *testing.T) {
 
 			response := &http.Response{
 				Request: request,
-				Body: ioutil.NopCloser(bytes.NewReader(bodyBytes)),
+				Body:    ioutil.NopCloser(bytes.NewReader(bodyBytes)),
```

Comment:
Sorry for the sloppy commit.  I've run `go fmt …` and squashed the fixes in [here][1].